src/notation/notationtypes.h

```diff
@@ -136,6 +136,7 @@ using InstrumentGenreList = QList<const InstrumentGenre*>;
 using ScoreOrderList = QList<const ScoreOrder*>;
 using InstrumentGroupList = QList<const InstrumentGroup*>;
 using MidiArticulationList = QList<MidiArticulation>;
+using PartId = ID;
```

Comment:
I do not agree with this. Making aliases for objects is not a good idea because we have many objects and will have many aliases. Just use an id with a named argument

Reply:
There is a couple of cons of that approach, but in general I'm not against